src/Context/Doctrine/AbstractAliceContext.php

```diff
@@ -174,12 +174,39 @@ private function loadFixtures($fixturesFiles, $persister = null)
             $persister = $this->castServiceIdToPersister($persister);
         }
 
+        $fixtureBundles = [];
+        $fixtureDirectories = [];
+
         foreach ($fixturesFiles as $key => $fixturesFile) {
-            if (0 !== strpos($fixturesFile, '/') && 0 !== strpos($fixturesFile, '@')) {
+            if (0 === strpos($fixturesFile, '/')) {
+                if (is_dir($fixturesFile)) {
+                    $fixtureDirectories[] = $fixturesFile;
+                    unset($fixturesFiles[$key]);
+                }
+            } elseif (0 === strpos($fixturesFile, '@')) {
+                if (false === strpos($fixturesFile, '.')) {
+                    $fixtureBundles[] = $this->kernel->getBundle(substr($fixturesFile, 1));
+                    unset($fixturesFiles[$key]);
+                }
+            } else {
```

Comment:
We can't put condition in a single if and we still need to nest the if in order to not break the logic. If a fixtures file starts with a `/` or an `@` and is not a directory then, we must not use the base path. Anyway, I have split it in order to not rely on `if/elseif/else` :)